am-do-2-atom-do.py

```diff
@@ -0,0 +1,410 @@
+    duration = script_end - script_start
+    print("Script duration: " + str(duration))
+    print("Number of errors encountered: " + str(ERROR_COUNT))
+
+
+def flush_legacy_digital_file_properties(legacy_dip_files):
+    global ERROR_COUNT
+
+    for file in legacy_dip_files:
+        try:
+            # Select the Archivematica Object UUID which will be used to find
+            # property info in the METS file.
+            sql = "SELECT value FROM property_i18n WHERE id = %s;"
+            mysqlCursor.execute(sql, file['id'])
+            object_uuid = mysqlCursor.fetchone()
+        except Exception as e:
+            print("Unable to select Object UUID for object# " + str(file['id']) + ". Skipping...")
+            print(e)
+            ERROR_COUNT += 1
+            continue
+
+        try:
+            # Select the Archivematica AIP UUID which will be used to fetch
+            # the METS file from the Storage Service.
+            sql = "SELECT id from property WHERE name = 'aipUUID' AND object_id = %s;"
+            mysqlCursor.execute(sql, file['object_id'])
+            property_id = mysqlCursor.fetchone()
+            sql = "SELECT value FROM property_i18n WHERE id = %s;"
+            mysqlCursor.execute(sql, property_id['id'])
+            aip_uuid = mysqlCursor.fetchone()
+        except Exception as e:
+            print("Unable to select AIP UUID for object# " + str(file['id']) + ". Skipping...")
+            print(e)
+            ERROR_COUNT += 1
+            continue
+
+        try:
+            # Store identifier values in the working table.
+            sql = "INSERT INTO dip_files (object_id, object_uuid, aip_uuid, parsed) VALUES (%s, %s, %s, %s);"
+            mysqlCursor.execute(sql, (file['object_id'], object_uuid['value'], aip_uuid['value'], False))
+            mysqlConnection.commit()
+        except Exception as e:
+            print("Unable to insert working data for object# " + str(file['id']) + ". Skipping...")
+            print(e)
+            ERROR_COUNT += 1
+            continue
+
+        try:
+            # Flush the existing properties for the legacy digital file.
+            # Includes an automatic cascade delete of the i18n value.
+            # These properties will be replaced with values from the METS file.
+            sql = "DELETE FROM property WHERE object_id = %s;"
+            mysqlCursor.execute(sql, file['object_id'])
+            mysqlConnection.commit()
+        except Exception as e:
+            print("Unable to flush existing property values for object# " + str(file['id']) + ". Skipping...")
+            print(e)
+            ERROR_COUNT += 1
+            continue
+
+    return
+
+
+def get_mets_path(aip_uuid):
+    request_url = STORAGE_SERVICE_URL + "file/" + aip_uuid + "?username=" + STORAGE_SERVICE_USER + "&api_key=" + STORAGE_SERVICE_API_KEY
+    try:
+        response = requests.get(request_url)
+    except Exception as e:
```

Comment:
Can you be more specific? `except Exception` is preferable to a bare `except` but even better is to name the exception you're expecting. We're just calling `requests.get` here, so I'd suggest:

```suggestion
    except requests.exceptions.RequestException as e:
```